app/assets/javascripts/ama_validators/country_province_selector.js.coffee.erb

```diff
@@ -0,0 +1,33 @@
+class window.CountryProvinceSelector
+  constructor: (@country_select, @province_select) ->
+    _province_select = @province_select
+    _provinces = JSON.parse('<%= ProvinceValidator::PROV_STATE.to_json %>')
```

Comment:
In Membership, we have a separate file to load all the constants in, so only that 1 file needs to be an `.erb`:

https://github.com/amaabca/membership/blob/master/app/assets/javascripts/constants.coffee.erb